tracing.go

```diff
@@ -0,0 +1,48 @@
+// Copyright 2023 Heath Stewart.
+// Licensed under the MIT License. See LICENSE.txt in the project root for license information.
+
+package azcrypto
+
+import (
+	"context"
+
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore/tracing"
+)
+
+func (client *Client) startSpan(ctx context.Context, name string) (context.Context, span) {
```

Comment:
Good question.  Since this is not an official SDK then you don't _have_ to use `runtime.StartSpan`.  And since this SDK isn't making its own HTTP calls but calling into other SDKs then perhaps what you have is sufficient.